django_opensearch_dsl/registries.py

```diff
@@ -79,6 +85,42 @@ def register_document(self, document):
 
         return document
 
+    def _get_related_doc(self, instance):
+        for model in self._related_models.get(instance.__class__, []):
+            for doc in self._models[model]:
+                if instance.__class__ in doc.django.related_models:
+                    yield doc
+
+    def update_related(self, instance, **kwargs):
+        """Update docs that have related_models."""
+        if not DODConfig.autosync_enabled():
+            return
+
+        for doc in self._get_related_doc(instance):
+            doc_instance = doc()
+            try:
+                related = doc_instance.get_instances_from_related(instance)
+            except ObjectDoesNotExist:
+                related = None
+
+            if related is not None:
+                doc_instance.update(related, **kwargs)
+
+    def delete_related(self, instance, **kwargs):
+        """Remove `instance` from related models."""
```

Comment:
Can you detail the interaction with `related_instance_to_ignore` within the docstring ?

Reply:
Added quite a bit to this docstring. Let me know if there needs to be better clarification on that somewhere.